test/uk/gov/hmrc/hec/testonly/controllers/TaxCheckControllerSpec.scala

```diff
@@ -35,18 +36,19 @@ import uk.gov.hmrc.hec.testonly.services.TaxCheckService
 import uk.gov.hmrc.hec.testonly.models.SaveTaxCheckRequest
 import uk.gov.hmrc.hec.util.TimeUtils
 import uk.gov.hmrc.http.HeaderCarrier
+
 import java.time.LocalDate
 import java.time.format.DateTimeFormatter
-
 import scala.concurrent.Future
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class TaxCheckControllerSpec extends ControllerSpec {
+class TaxCheckControllerSpec extends ControllerSpec with AuthSupport {
```

Comment:
`AuthSupport` isn't being used here so we can clean this up here I think

Reply:
thanks for spotting, updated - https://github.com/hmrc/hec/pull/18/commits/8b7b7adb39a7ea2888b8cf693cf87cb5c2065f60